gitbase.go

```diff
@@ -11,10 +11,15 @@ func NewToolGitbase() regression.Tool {
 		GitURL:      "https://github.com/src-d/gitbase",
 		ProjectPath: "github.com/src-d/gitbase",
 		BuildSteps: []regression.BuildStep{
+			{
```

Comment:
this build step is unnecessary if `GO111MODULE=on` is present (see previous comment in travis.yml)

Reply:
Done